src/view/new-point.js

```diff
@@ -0,0 +1,344 @@
+import dayjs from 'dayjs';
+import he from 'he';
+import { TYPES_OFFERS, CITIES, DESCRIPTION_CITY, OFFERS, DEFAULT_TIME_DIFFERENCE } from './common/const.js';
+import { findOffersType, formatDateForEditPoint } from './utils/point.js';
+import { generatePhoto, getRandomDescription, generateDestination } from './mocks/point.js';
+import SmartView from './smart.js';
+import flatpickr from 'flatpickr';
+import '../../node_modules/flatpickr/dist/flatpickr.min.css';
+
+const typeBlank = TYPES_OFFERS[0];
```

Comment:
```suggestion
const [typeBlank] = TYPES_OFFERS;
```
Меньше магических значений